packages/commonwealth/client/scripts/models/ChainInfo.ts

```diff
@@ -172,6 +175,10 @@ class ChainInfo {
       : base === ChainBase.CosmosSDK
       ? 6
       : 18;
+
+    // TODO: this is temporary until we have a better way to handle governance versions
+    // see: https://github.com/hicommonwealth/commonwealth/issues/3292
+    const governance_version = id === 'kyve' ? 'v1' : 'v1beta1';
```

Comment:
@jnaviask  @CowMuon  Is this an ok place for this hard-code for now? I mention the line in #3292

Reply:
Thought about this more. I will put it in an env variable for now. That way we can add communities if needed. I'll use a comma-separated flag, like with the chain events flag.

Reply:
Did this ^ . Now we will just need to add vars to each env.
`COSMOS_GOV_V1=kyve`